VERSION

```diff
@@ -1 +1 @@
-3.2.7
\ No newline at end of file
+3.3.0
```

Comment:
Hey, thanks for the contribution!  I've re-read our CONTRIBUTING.md, and it's a bit misleading - in fact, our Continuous Integration system is in charge of all version increments, so this particular change should be reverted.

I've opened #3692 to update the CONTRIBUTING.md.

Reply:
Thanks!  Will do!